init.lua

```diff
@@ -58,59 +63,69 @@ minetest.register_chatcommand("watch", {
 	privs = {watch = true},
 	func = function(name_watcher, name_target)
 		if name_watcher == name_target then
-			return true, "You may not watch yourself"
+			return true, "You may not watch yourself."
 		end
 
 		local target = minetest.get_player_by_name(name_target)
 
 		if not target then
-			return true, "Unknown target player name"
+			return true, 'Invalid target name "' .. name_target .. '"'
 		end
 
 		-- avoid infinite loops
-		if original_pos[name_target] then
-			return true, name_target .. " is already watching a player."
+		if original_state[name_target] then
+			return true, '"' .. name_target .. '" is watching "'
+				.. original_state[name_target].target .. '". You may not watch a watcher.'
```

Comment:
Not a change request, but a suggestion in case you prefer that style:

```suggestion
			return true, ('"%s" is watching "%s". You may not watch a watcher.'):format(
				name_target, original_state[name_target].target)
```

Reply:
yep, I prefer this style as it is less prone to bugs from missing arguments or wrong sort order.
Once locales are added this will change things again.